app/components/ui/molecules/PeoplePicker.md

```diff
@@ -0,0 +1,49 @@
+`PeoplePicker` uses a render prop to allow customising the layout.
+
+### Buttons above body
+
+```js
+const people = [
+  { id: 1, name: 'Annie Badger', institution: 'A University' },
+  { id: 2, name: 'Bobby Badger', institution: 'B College' },
+  { id: 3, name: 'Chastity Badger', institution: 'C Institute' },
+  { id: 4, name: 'Dave Badger', institution: 'D Research Lab' },
+]
+;<PeoplePicker
+  minSelection="2"
+  maxSelection="3"
+  onSubmit={selection => console.log(selection)}
+  people={people}
+>
+  {props => (
+    <React.Fragment>
+      <PeoplePicker.Buttons {...props} />
+      <hr />
+      <PeoplePicker.Body {...props} />
+    </React.Fragment>
+  )}
+</PeoplePicker>
+```
+
+### Body only
```

Comment:
Just looking at this component in styleguidist, before looking at the code, this combination of props doesn't really show everything that the `Body` component can do & is responsible for.

Right now, it looks like the icon buttons are always disabled in "Body", which isn't the case, and therefore people don't get to see that "Body" is also responsible for showing/hiding the `SelectedItem` tags & the `SelectionHint` message.

Perhaps we add `isSelected` & `toggleSelection` handlers that change state in the same way that we do in our app, in order to demonstrate this functionality?